hugo/content/docs/ast-types.md

```diff
@@ -225,7 +225,13 @@ left_right --> left_right_{2}
 {{</mermaid>}}
 
 ## Declared Types
-Because type inference takes into account every entity of a parser rule, even the smallest changes can update the inferred types. This can lead to an unexpected type system and incorrect behavior of services that depend on it. To minimize the risk of introducing breaking changes when modifying the grammar, we recommend to use *declared types*. This is especially true for more mature and complex languages, where a stable type system is key and breaking changes introduced by inferred types can be hard to detect. Declared types allow the user to **fix** the type of the parser rules and rely on the power of validation errors to detect breaking changes.
+Because type inference takes into account every entity of a parser rule, even the smallest changes can update your inferred types. This can lead to an unexpected type system and incorrect behavior of services that depend on it. To minimize the risk of introducing breaking changes when modifying the grammar, we have introduced *declared types* as a new feature.
```

Comment:
> This can lead to an unexpected type system and incorrect behavior of services that depend on it.

This can lead to an unwanted changes of the semantic model and incorrect behavior of services that depend on it.